package.json

```diff
@@ -1,25 +1,25 @@
 {
   "name": "VoIPGRID",
-  "version": "1.0.5",
+  "version": "1.0.6",
   "description": "Zapier integration with the VoIPGRID platform.",
   "main": "index.js",
   "scripts": {
     "test": "node node_modules/mocha/bin/mocha --recursive -t 10000"
   },
   "engines": {
-    "node": ">=6.10.0",
-    "npm": ">=3.0.0"
+    "node": ">=12.13.0",
+    "npm": ">=6.14.4"
   },
   "dependencies": {
-    "lodash": "4.17.11",
-    "zapier-platform-cli": "^8.1.0",
-    "zapier-platform-core": "8.1.0"
+    "lodash": "^4.17.15",
+    "zapier-platform-cli": "10.0.0",
```

Comment:
The [zapier-platform CHANGELOG](https://github.com/zapier/zapier-platform/blob/master/CHANGELOG.md) suggests that their might be some breaking changes?

Reply:
Ik heb de changelog doorgelezen, en zie daar niks breaking in staan. Tenminste niet in deze koppeling.